doc/python/mechanisms.rst

```diff
@@ -126,14 +126,14 @@ Cable cell mechanisms
         The underlying mechanism.
 
     .. method:: density(name)
-        :noindex:
 
         constructs :attr:`mech` with *name* and default parameters.
 
         :param name: name of mechanism.
         :type name: str
 
     .. method:: density(name, params)
+        :noindex:
```

Comment:
Why is it here anyway?

Reply:
I moved the `noindex` attribute from the first method overload to the second just to be consistent.

Reply:
Sure, I guess for crossref within the docs that's good enough.